packages/atomic/src/locales.json

```
@@ -612,6 +612,14 @@
     "zh-CN": "没有找到与 {{query}} 相关的结果",
     "zh-TW": "沒有找到與 {{query}} 相關的結果"
   },
+  "showing-results-for": {
+    "en": "Showing results for {{query}}",
+    "fr": "Résultats affichés pour {{query}}"
+  },
+  "search-instead-for": {
+    "en": "Search instead for {{query}}",
+    "fr": "Rechercher à la place {{query}}"
```

```suggestion
    "fr": "Rechercher {{query}} à la place"
```

---

If {{query}} must stay at the end, do you think it would be possible to go:
```suggestion
    "fr": "Rechercher plutôt {{query}}"
```
Not a dealbreaker, but it's grammattically better 🙂